source/playgrounds.txt

```diff
@@ -150,19 +150,41 @@ If you have a playground file open in |vscode-short| and do not have
 an active connection, |vsce| displays
 :guilabel:`Click here to add connection` at the top of your playground.
 
-.. figure:: /images/vsce-playground-add-connection-btn.png
-   :figwidth: 700px
-   :alt: Link to add connection from playground
-
-1. Click this link to open the connection string dialog.
-
-#. Enter the
-   :manual:`connection string </reference/connection-string/>` for the
-   deployment you want to run this playground against.
+.. procedure::
+   :style: normal
    
-#. Press :guilabel:`Enter`.
-
-#. Run your playground.
+   .. step:: Select :guilabel:`Click here to add connection` 
+
+      After you click :guilabel:`Click here to add connection`, |vsce| opens the 
+      connection string dialog.
+
+      .. figure:: /images/vsce-playground-add-connection-btn.png
+         :figwidth: 700px
+         :alt: Link to add connection from playground
+         
+   .. step:: Enter your connection string
+      
+      Enter the :manual:`connection string </reference/connection-string/>` for 
+      the deployment you want to run this playground against.
+
+      .. note:: 
+         
+         If the connection string specifies a database, the playground runs 
+         against the default connected database unless you explicitly call 
+         ``use('<database_name>')``  to use another database.
```

Comment:
```suggestion
         against the default connected database unless you explicitly switch databases with 
         ``use('<database_name>')``.
```

Reply:
Split this into two sentences. LMK what you think.